ostelco-core/Network/Request.swift

```diff
@@ -47,6 +47,7 @@ public struct Request {
         
         var request = URLRequest(url: url)
         request.httpMethod = self.method.rawValue
+        request.timeoutInterval = 180.0
```

Comment:
do we want every request to have a 5 minute timeout, or just some requests?

Reply:
This was just an attempt to get something ready for regulator meeting. We shouldn't keep it.
There's only one endpoint that can take > 60 seconds (observed both in prod and dev) and that is this endpoint `GET /regions/sg/kyc/myInfo/{authorisationCode}`

I really want to figure out why it can take that long, before we `solve` it by increasing the request timeout

Reply:
cool - maybe revert this for the time being then